README.md

```diff
@@ -49,17 +49,24 @@ Here's the current matrix of supported features per platform:
 
 ¹: Apple currently includes:
 
-- `macosX64`
-- `macosArm64`
-- `iosArm64`
-- `iosX64`
-- `iosSimulatorArm64`
-- `watchosArm32`
-- `watchosArm64`
-- `watchosSimulatorArm64`
-- `tvosArm64`
-- `tvosX64`
-- `tvosSimulatorArm64`
+* `macosX64`
+* `macosArm64`
+* `iosArm64`
+* `iosX64`
+* `iosSimulatorArm64`
+* `watchosArm32`
+* `watchosArm64`
+* `watchosSimulatorArm64`
+* `tvosArm64`
+* `tvosX64`
+* `tvosSimulatorArm64`
```

Comment:
Out of curiosity, what is your linter? We typically edit this file with IntelliJ so I'd like to commit the markdown rules like we have for [Kotlin](https://github.com/apollographql/apollo-kotlin/blob/main/.idea/codeStyles/Project.xml)